bin/init.bash

```diff
@@ -328,7 +328,8 @@ backup_file() {
 
   log_info "Creating backup ${backup_config_path}/${backup_name}"
 
-  cp "${file}" "${backup_config_path}/${backup_name}"
+  # --force to workaround issue in integration test where file already exists
+  cp --force --backup "${file}" "${backup_config_path}/${backup_name}"
```

Comment:
Is this still needed, or was it because it ran multiple ones concurrently?

Reply:
Will check 🤔

Reply:
Tests pass with that reverted, so doesn't seem necessary anymore